drivers/scheduler/k8s/k8s.go

```diff
@@ -830,6 +868,21 @@ func (k *K8s) substituteNamespaceInPVC(pvc *v1.PersistentVolumeClaim, ns string)
 	}
 }
 
+func (k *K8s) addVpsVolumeLabel(pvc *v1.PersistentVolumeClaim, vlabel string) {
+	//pvc.Metadatas.Labels["appvps"] = vlabel
+	logrus.Infof("PVC label update label:%v  pvc:%v", vlabel, pvc)
+}
+
+//Set or disable the placement_strategy in the storageclass
+func (k *K8s) substituteVpsNameInStorageClass(sc *storageapi.StorageClass, vpsname string) {
+	//	sc.Name = namespaceRegex.ReplaceAllString(sc.Name, ns)
+	if sc.Parameters["aggregation_level"] == "2" || sc.Parameters["aggregation_level"] == "3" {
```

Comment:
why are we doing this? We do support placement_strategy for aggregation_level 2 and 3. If we don't want to test

placement for aggregation_level = 2 or 3 for a reason, we need to handle that at a spec level and not here